.changeset/two-paws-double.md

```diff
@@ -0,0 +1,6 @@
+---
+'wingman-fe': minor
```

Comment:
```suggestion
'wingman-fe': major
```

I know it's silly, but it is accurate if we're bumping a peer dep requirement

Reply:
Yeah you're right, I always mess this up 😅